savant/deepstream/pipeline.py

```diff
@@ -948,6 +940,26 @@ def _create_muxer(self, live_source: bool) -> Gst.Element:
 
         return self._muxer
 
+    def _create_muxer_config(self) -> str:
+        file = tempfile.NamedTemporaryFile(
+            prefix='nvstreammux_',
+            suffix='.txt',
+            delete=False,
+        )
+        params = {
+            'algorithm-type': 1,
+            'batch-size': self._batch_size,
+            'max-same-source-frames': self._batch_size,
```

Comment:
Default value is ok, but please add an option to limit that.

Reply:
Added related configs